### PR TITLE
Remove WelcomeToNewVAModal

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/8499

This PR removes the `<WelcomeToNewVAModal />`.

## Testing done
Tests passing locally 🎉 

## Screenshots
N/A

## Acceptance criteria
- [x] Remove the `<WelcomeToNewVAModal />`.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/announcements/config/index.js [hunk 1]
```diff
@@ -59,11 +58,6 @@ const config = {
       showEverytime: true,
       relatedAnnouncements: ['welcome-to-new-va'],
     },
-    {
-      name: 'welcome-to-new-va',
-      paths: /^\/$/,
-      component: WelcomeToNewVAModal,
-    },
```
Comment (ncksllvn) [inline src/platform/site-wide/announcements/config/index.js:66]: Not super necessary but we could also remove the reference to it on line 59

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/platform/site-wide/announcements/config/index.js (hunk 1 above)
Comment (kelsonic) [reply src/platform/site-wide/announcements/config/index.js:66]: Ah yes! Great catch!

**Comment by rianfowler** [comment]
awesome

**Pull request merged**